spacepy/pybats/rim.py

```diff
@@ -387,7 +365,7 @@ def add_cont(self, var, target=None, n=50, maxz=False, lines=False,
             that is strictly positive and "Seismic" for diverging data.
             Alternatively, legacy Ridley Ionosphere Model color maps can be
             loaded using "l_wr" (white red) or "l_bwr" (blue-white-red),
-            where the "l_" prefix indicates legacy and not Matplotlib color maps.
+            where the ``"l_"`` prefix indicates legacy and not Matplotlib color maps.
```

Comment:
Ugh, this gets nasty...can we just make them all monospace with no quotes for consistency here? Or given that we have the others up there, just leave the `l_` in normal type in double quotes, but escape the underscore? `"l\_"`